src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -252,59 +186,29 @@ must be supplied as the value to be matched."
   {:pre [(string? table)
          (string? column)]}
   (jdbc/with-db-transaction []
-   (if (postgres?)
-     ;; PostgreSQL specific way
-     (do
-       (jdbc/do-commands (str "LOCK TABLE " table " IN ACCESS EXCLUSIVE MODE"))
-       (jdbc/query-with-resultset
-        [(str "SELECT setval(pg_get_serial_sequence(?, ?),
-                             (SELECT max(" column ") FROM " table "))")
-         table column]
-        (constantly nil)))
-
-     ;; HSQLDB specific way
-     (let [_ (jdbc/do-commands (str "LOCK TABLE " table " WRITE"))
-           maxid (jdbc/query-with-resultset
-                  [(str "SELECT max(" column ") as id FROM " table)]
-                  (comp :id first sql/result-set-seq))
-           ;; While postgres handles a nil case gracefully, hsqldb does not
-           ;; so here we return 1 if the maxid is nil, and otherwise return
-           ;; maxid +1 to indicate that the next number should be higher
-           ;; then the current one.
-           restartid (if (nil? maxid) 1 (inc maxid))]
-       (jdbc/do-commands (str "ALTER TABLE " table " ALTER COLUMN " column
-                              " RESTART WITH " restartid))))))
+    (jdbc/do-commands (str "LOCK TABLE " table " IN ACCESS EXCLUSIVE MODE"))
+    (jdbc/query-with-resultset
+     [(str "SELECT setval(pg_get_serial_sequence(?, ?),
+                          (SELECT max(" column ") FROM " table "))")
```

Comment:
I think the string can be broken up a little more so that there isn't all the extra whitespace in the query.